src/phaser-game/scenes/SingleMode.ts

```diff
@@ -0,0 +1,273 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import Phaser from 'phaser';
+import {
+  BULLET_MOVEMENT,
+  BULLET_OFFSET,
+  NPC_DIMENSIONS,
+} from '../utils/constants';
+import npcData from '../../NPC.json';
+
+// let background: any;
+let player: any;
+let cursor: any;
+let collidableObjects: any;
+let shootBullet: any;
+let bullet: any = {};
+let speed: number;
+let score = 0;
+let scoreText: any;
+let npc: any;
```

Comment:
I may have mentioned this previously, but please work towards using more specific non-`any` types for your variables.